topcoffea/modules/histEFT.py

```diff
@@ -318,6 +317,56 @@ def __reduce__(self):
             ),
         )
 
+    def make_scalings(self,h,ch,p):
```

Comment:
Why are you passing a histogram `h` into this function.  Shouldn't this function just operate on `self`?